components/event-sources/Makefile

```diff
@@ -103,7 +108,7 @@ ISTIO_API_GROUPS := "security/v1beta1"
 # generates e.g. "ISTIO_CLIENT_GO_PKG/apis/sources/v1alpha1 ISTIO_CLIENT_GO_PKG/apis/sources/v1alpha2"
 istio-api-import-paths := $(foreach group,$(ISTIO_API_GROUPS),$(ISTIO_CLIENT_GO_PKG)/apis/$(group))
```

Comment:
Are we still using `istio-api-import-paths` and `ISTIO_API_GROUPS `?

Reply:
I don't know I am not maintaining this component.